### PR TITLE
adding exposure events to biolink

### PR DESCRIPTION
**Pull request by diatomsRcool**
I think this will change things for golr, but I'm not familiar enough with how that works to know what's best. #215

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -2875,6 +2875,32 @@ classes:
     description: >-
       An association between a biosample and a disease or phenotype
 
+  disease to exposure association:
+    is_a: association
```
[the remark this answers is not in the document]
Comment (diatomsRcool) [reply biolink-model.yaml]: fixed

File: biolink-model.yaml [hunk 2]
```diff
@@ -2875,6 +2875,32 @@ classes:
...
+      - object
+    description: >-
+      An association between an exposure event and a disease
+    slot_usage:
+      subject:
+        range: disease
+      object:
+        range: exposure event
+
+  phenotype to exposure association:
```
[the remark this answers is not in the document]
Comment (diatomsRcool) [reply biolink-model.yaml]: I actually found an existing association "environment to phenotypic feature" this is duplicative so I removed it and modified the existing association.

**Comment by diatomsRcool** [comment]
What should become of "chemical to disease or phenotypic feature association"?

File: biolink-model.yaml [hunk 3]
```diff
@@ -2875,6 +2875,32 @@ classes:
...
+    is_a: association
+    defining_slots:
+      - subject
+      - object
+    description: >-
+      An association between an exposure event and a phenotype
+    slot_usage:
+      subject:
+        range: phenotype
```
[the remark this answers is not in the document]
Comment (diatomsRcool) [reply biolink-model.yaml]: I added receptor, stressor, and route as node properties for exposure event

**Comment by deepakunni3** [comment]
@diatomsRcool I am going to go ahead and remove the following from the tests folder:
- `tests/biolink/*`
- `tests/model.py`
- `tests/biolink-model.yaml`.

These are never used for running tests. Or maybe they were before, but not anymore.

@hsolbrig Let me know if the above three files in the tests folder are needed.

File: biolink-model.yaml [hunk 4]
```diff
@@ -2518,15 +2547,29 @@ classes:
           - value: ClinVarVariant:17681
             description: NM_007294.3(BRCA1):c.2521C>T (p.Arg841Trp)
 
+  chemical exposure:
+#    aliases: ['drug intake', 'drug dose']
+    is_a: exposure event
+    description: >-
+      A chemical exposure is an intake of a particular chemical substance
+#    slots:
```
[the remark this answers is not in the document]
Comment (diatomsRcool) [reply biolink-model.yaml]: They can be removed as far as i'm concerned.

**Comment by hsolbrig** [comment]
They are not.  Good find.


Harold Solbrig

**Comment by deepakunni3** [comment]
Thanks for confirming :+1:

**Pull request merged**